src/scripts/main.js

```diff
@@ -942,6 +943,43 @@
                     }, window.origin);
                 }
             }
+            else if (markup.render_data.entry_timestamp === live_ts && 
+                css_class.search(/alchemists_cookbook_base_bonus/) !== -1) {
+
+                if (markup.render_data.entry_timestamp === live_ts) {
+                    more_details['alchemists_cookbook_base_bonus'] = true;
+                    if (debug_logging) {window.console.log({procs: more_details});}
+                }
+            }
+            else if (markup.render_data.entry_timestamp === live_ts && 
+                    css_class.search(/boiling_cauldron_trap_bonus/) !== -1) {
+                const data = markup.render_data.text;
+                const potionRegex = /item\.php\?item_type=(.*?)"/; 
+                if (potionRegex.test(data)) {
+                    const resultPotion = data.match(potionRegex)[1].trim();
+                    if ("inventory" in hunt_response && resultPotion in hunt_response.inventory) {
+                        const potionName = hunt_response.inventory[resultPotion].name;
+                        const potionId = hunt_response.inventory[resultPotion].item_id;
+                        if (potionName && potionId) {
+                            const convertible = {
+                                id: 3304, // Boiling Cauldron Trap
+                                name: "Boiling Cauldron Trap",
+                                quantity: 1,
+                            };
```

Comment:
We should have a single place where these "known" convertibles get stored and retrieved from.

Reply:
Agree. This is another of those weird ones like Riftios but this one has an item id from mousehunt so I used that. I could have put `user.weapon_item_id` instead -- if we confirmed the right weapon was armed or whatever other suspenders we wanted to add to it.